FrontEnd/.env-default

```diff
@@ -1 +1,3 @@
 REACT_APP_INSTRUMENTATION_KEY=
+REACT_APP_SEPES_LOGIN_URL=http://localhost:5000/api/auth/token
```

Comment:
Backend forces HTTPS so these would just be redirected immediately. Might as well just start the request with https